packages/block-library/src/code/theme.scss

```diff
@@ -1,6 +1,10 @@
+.wp-block-code {
+	border: 1px solid #ccc;
+	border-radius: 4px;
```

Comment:
Since we have the `border-radius` support, should we default to `zero`?

Reply:
Long term, I would like to see every `theme.scss` retired and replaced by theme.json powered presets, provided either by the editor, the theme, or both. A classic example is showing a block quote with a thin border on the left, this should ideally be a theme.json defined preset, and not be based on CSS from these files.

Until we can reach that point, though, my personal instinct is to keep these styles mostly as they are, for back compat purposes. In this case, keeping the radius as is. Thankfully you can explicitly set it to zero, or anything you like, using both theme.json or the UI control. What do you think?